prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml

```diff
@@ -0,0 +1,139 @@
+presubmits:
+  IBM/ibm-commonui-operator:
+  - name: build_ibm-commonui-operator
+    branches:
+    - ^release-1.5$
+    decorate: true
+    always_run: true
+    path_alias: github.com/IBM/ibm-commonui-operator
+    rerun_command: /test build ibm-commonui-operator
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - make
+        - build
+        image: quay.io/multicloudlab/build-tool:v20200817-b1f2b4c05
```

Comment:
@siddarthpatel Any reason why we don't use the image in the master branch?

Reply:
The image used for the master branch includes `go version  1.15.7` can avoid some vulnerability issues.

Reply:
That was a mistake by me. I update to use the images in master branch